chapters/en/slides/module2/module2_29.Rmd

```diff
@@ -129,20 +143,24 @@ Notes: Script here
 
 ---
 
-We've added a title before, so there is nothing new there but adding x and y-axis labels is a little different. We can increase the label font sizes using the argument `fontsize`. In this case, we reference our initial plot and use the verb `.set_ylabel()` and `.set_xlabel()` with the desired axis label as an argument and `fontsize` to assign a desired label size. 
-To avoid unnecessary information that will be returned otherwise, whatever our last verb being used with our plot (named `sugar_plot) has to be reassigned back to the object. If we did this any other way, we would not have the ability to do more transformations on our plot, or we would get additional information with the plot output. 
-
-```{python  out.width = '45%', fig.asp = .40}
-sugar_plot = (df.groupby(by='mfr')
-                .mean()
-                .loc[:,'sugars']
-                .plot.bar(title='Mean sugar content among manufacturers')
-              )
-sugar_plot.set_ylabel('Sugar content (in grams)', fontsize=9)
-sugar_plot = sugar_plot.set_xlabel('Manufacturer', fontsize=9)
-sugar_plot
+Another interesting thing we can do with this new syntax is change the number of `bins` when generating a bar plot. Having control over the number of bins in a bar plot may help to make visualization easier which will make the plots easier to digest.  If we wanted to change the number of bins, we can use the `bin=alt.Bin(maxbins=..)` in `alt.X()` to set a value for the number of bins. For example, we can generate a bar plot of the `calories` column in the cereal dataframe and set the number of bins to `50` by setting `bin=alt.Bin(maxbins=50)`.
```

Comment:
Before this, we need to show it failing if we don't do this. We discussed changing the mfr column to a number or something. This needs an example before going forward.

Reply:
Okay, will think of something!